WordPress/Classes/Extensions/NSString+Helpers.swift

```diff
@@ -19,4 +21,14 @@ extension NSString
         
         return uniqueSet
     }
+    
+    /// Validates the current string. Returns true if passes validation
+    ///
+    public func isValidEmail() -> Bool {
+        // From http://stackoverflow.com/a/3638271/1379066
+        let emailRegex = "^[^@\\s]+@[^@\\s]+$"
```

Comment:
An inusual-but-valid address that fails to pass this test:

`"Jorge Bernal"@example.com`

We can keep playing this game for a while 😏